build/distributions/CentOS/7/x86_64/DVD/ks/dvd/auto.cfg

```diff
@@ -4,7 +4,7 @@ text
 keyboard us
 lang en_US
 
-%include /mnt/install/repo/ks/dvd/include/common_ks_base
+%include /mnt/stage2/ks/dvd/include/common_ks_base
```

Comment:
Actually, I'm reverting this entire file. There are no changes that need to be made from what I can see and many changes that I'm a bit surprised worked.